Runtime/McSubscriber.cs

```diff
@@ -508,6 +553,35 @@ public void RemoveRenderAudioSource(AudioSource source)
         {
             _renderer.RemoveAudioTarget(source);
         }
+        private Layer GetSimulcastlayer(string rid)
```

Comment:
Need to follow proper pascal naming convention here

Reply:
done.